multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -84,20 +86,24 @@ def is_categorical_column(
                 ratio = 0.1
         threshold = min(int(len(data) * ratio), threshold)
         data_value_counts = data.value_counts(dropna=False)
```

Comment:
I believe this call also fails if `data` is a bytearray because `value_counts` requires hashing the entries (AFAIK).

Reply:
It didn't error out during test, but I agree it would be safer to include this inside the `try` as well. Thanks for pointing it out!